### DISCUSSION TITLE
What the use of picture named 12h.gif in rdkit/utils

### DISCUSSION THREAD

**Discussion by Joezczhang** [rdkit/rdkit#4437] [Q&A] [answered]
I'm very confused about picture name 12h.gif in rdkit/utils, is there some introduction about it?

**Comment by greglandrum** [comment]
No, I'm not quite sure how that ended up in the repo, but it's been there for a very long time.
Thanks for pointing out the bogus file! I'll remove it.

**Comment by Joezczhang** [comment, marked as answer]
Thanks a lot.
